joi-validations/topics.validations.js

```diff
@@ -5,6 +5,16 @@ const TopicValidation = {
     query: {
       name: string.required()
     }
+  },
+  checkName: {
```

Comment:
@brm-stnd @ibramastana please confirm with product team, this should have minimum query length. We don't want to search the DB with name less than 3 since that will be heavy.

Reply:
ok baik mas, sudah saya diskusikan sm mas gerry, kami selakat minimal 3 char